build_tools/azure/windows.yml

```diff
@@ -27,6 +27,12 @@ jobs:
     - bash: python build_tools/azure/get_selected_tests.py
       displayName: Check selected tests for all random seeds
       condition: eq(variables['Build.Reason'], 'PullRequest')
+    - bash: echo "##vso[task.prependpath]$CONDA/Scripts"
+      displayName: Add conda to PATH
+      condition: startsWith(variables['DISTRIB'], 'conda')
+    - bash: build_tools/azure/install_setup_conda.sh
+      displayName: Install conda if necessary and set it up
+      condition: startsWith(variables['DISTRIB'], 'conda')
```

Comment:
adding conda to PATH is also done in `install_setup_conda.sh`, so we probably don't need it anymore if we're using the script.

Wouldn't it make things simpler to put the accept tos command there ?
https://github.com/scikit-learn/scikit-learn/blob/fe6960bac60a600479ea6d65f28a03115be7364c/build_tools/azure/install.sh#L57-L60